sympy/polys/ring_series.py

```diff
@@ -439,10 +431,10 @@ def _series_inversion1(p, x, prec):
         prec = int(prec)
 
     if zm not in p:
-        raise ValueError('no constant term in series')
+        raise ValueError("No constant term in series")
     if _has_constant_term(p - c, x):
-        raise ValueError('p cannot contain a constant term depending on '
-            'parameters')
+        raise ValueError("p cannot contain a constant term depending on "
+                         "parameters")
```

Comment:
@toolforger 
> Not just in SymPy, but in the Python community in general, though of course everybody's >perception tends to be different due to different peer groups. All I can say that in the >discussions I've seen on the topic, more often than not, hanging indents are considered an >undesirable option, some people even consider their inclusion in PEP8 a mistake; more >importantly, I see little if any protest against that stance. In fact you're the first person I ever >met who expressed a strong preference for hanging indents.

I'm afraid I had assumed the opposite as I was asked to change from multiple-of-four indent to hanging indent in this PR (see the outdated diff above). That said, I don't have a strong preference for either of them. There is too much variety in coding styles (including hanging indents) even in Sympy.

Since this PR is already merged, you can make the changes in another PR if you want.